Mathlib.lean

```diff
@@ -101,6 +101,10 @@ import Mathlib.Algebra.Category.Ring.FilteredColimits
 import Mathlib.Algebra.Category.Ring.Instances
 import Mathlib.Algebra.Category.Ring.Limits
 import Mathlib.Algebra.Category.SemigroupCat.Basic
+import Mathlib.Algebra.Center.Basic
+import Mathlib.Algebra.Center.Ring
+import Mathlib.Algebra.Centralizer.Basic
+import Mathlib.Algebra.Centralizer.Ring
```

Comment:
Could you rename the files so that we avoid the explosion of the number of subfolders and instead keep the thematical `Algebra.Group` and `Algebra.Ring`?
```suggestion
import Mathlib.Algebra.Group.Centralizer
import Mathlib.Algebra.Ring.Centralizer
```

Reply:
Good suggestion, done!